tools/interactive/interactivetool_odv.xml

```diff
@@ -26,7 +26,7 @@
     ]]>
     </command>
     <inputs>
-        <param name="infile" type="data" format="netcdf" label="Netcdf file"/>
+        <param name="infile" type="data" format="netcdf,tabular" label="Netcdf file"/>
```

Comment:
The help needs to be adopted.

Reply:
Oh eagle eye Björn! You are totally right!!!